cli/flags.rs

```diff
@@ -298,6 +298,8 @@ pub struct Flags {
   pub future_type_check_mode: FutureTypeCheckMode,
   pub has_check_flag: bool,
   pub config_path: Option<String>,
+  // Disable loading of configuration file
+  pub no_config: bool,
```

Comment:
Nitpick: Maybe we could represent this in the struct as a single property in order to force it to be checked? Right now someone could access `config_path` and maybe forget to check `no_config`.

Reply:
I've added it, but that's a lot of additional code. Not sure if it's worth it.

Reply:
Looks good now with the enum 👍